src/camlsnark_c/bindings/vector.mli

```diff
@@ -1,4 +1,4 @@
-type 'a t
+type 'a t = unit Ctypes.ptr
```

Comment:
This makes me a little uncomfortable, since it makes all of the vector types alias to one another. Would it be a problem to make this opaque again?